articles/quickstart/native/_includes/ionic/_note_storage.md

```diff
@@ -0,0 +1,9 @@
+<!--markdownlint-disable MD041 -->
+
+:::note
+`localStorage` should be considered **transient** in a Capacitor app, as the operating system may recover disk space from `localStorage` if it is running low. Please read the [guidance on storage in the Capacitor docs](https://capacitorjs.com/docs/guides/storage#why-cant-i-just-use-localstorage-or-indexeddb).
+
+The underlying SDK has the ability to use a custom cache implementation to store tokens, if you have a requirement to use a more secure and persistent storage mechanism.
```

Comment:
```suggestion
The underlying SDK has the ability to use a custom cache implementation to store tokens, useful in case you have a requirement to use a more secure and persistent storage mechanism.
```

Reply:
Hmm not sure about this one, in the interest of conciseness 🤔

Reply:
Let's keep it.